src/modules/logger/logged_topics.cpp

```diff
@@ -64,20 +64,20 @@ void LoggedTopics::add_default_topics()
 	add_topic("estimator_status", 200);
 	add_topic("home_position");
 	add_topic("hover_thrust_estimate", 100);
-	add_topic("input_rc", 200);
+	add_topic("input_rc", 500);
```

Comment:
Why this? Does it not mean it's now harder to see tricky/weird RC related issues?

Reply:
I reduced it because `manual_control_setpoint` is no longer filtered. If we move `input_rc.rssi` I'd probably consider not logging it at all by default.

In general I want to try and reduce the bulk data were logging (by default) and prioritize status flags, errors, changes, etc. That's partially the motivation for things like https://github.com/PX4/Firmware/pull/15439 and https://github.com/PX4/Firmware/pull/15440.

For example if there's spurious garbage in the raw RC input we're already quite likely to miss it, but we should definitely catch any resulting state machine transitions and the reason (soon...).